OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
-    <string name="monitoring_min_accuracy_descr">This will record only points measured with a minimum "accuracy" indication (in meters/feet, as reported by Android for your chipset). Accuracy refers to the scatter of repeated measurements, and is not directly related to precision, which defines how close your measurements are to your "true" position.</string>
-    <string name="monitoring_min_accuracy_descr_side_effect">Side effect: As a result of filtering by accuracy, points may be entirely missing for e.g. below bridges, under trees, between high buildings, or with certain weather conditions.</string>
-    <string name="monitoring_min_accuracy_descr_recommendation">Recommendation: It is hard to predict what will be recorded and what not, it may be best to turn this filter off.</string>
-    <string name="monitoring_min_accuracy_descr_remark">Remark: If GPS had been off immediately before a recording, the first point measured may have a decreased accuracy,so in our code we may want to wait a second or so before recording a point (or record the best of 3 consecutive points, etc.), but this is not yet implemented.</string>
-    <string name="monitoring_min_distance_descr">This filter avoids duplicate points being recorded where too little actual motion may have occurred, makes a nicer spatial appearance of tracks not post-processed later.</string>
-    <string name="monitoring_min_distance_descr_side_effect">Side effects: Periods at rest are not recorded at all or by just one point each. Small (real world) movements (e.g. sideways, to mark a possile turnoff on your trip) may be filtered out. Your file contains less information for post-processing, and has worse stats by filtering out obviously redundant points at recording time, while potentially keeping artifacts caused by bad reception or GPS chipset effects.</string>
-    <string name="monitoring_min_distance_descr_recommendation">Recommendation: A setting of 5 meters may work well for you if you do not require to capture details finer than that, and do not want to explicitly capture data while at rest.</string>
+    <string name="monitoring_min_speed_descr">A low-speed cut-off filter smoothing recorded tracks by discarding track points below a certain speed.</string>
+    <string name="monitoring_min_speed_descr_side_effect">Sections (and breaks) below minimum speed are not recorded, like the steep hill you pushed your bike up. Total length, time in motion, and average speed can still be determined.</string>
+    <string name="monitoring_min_speed_descr_recommendation">Try using motion detection via the "Minimum logging displacement" filter (B) first. It may produce better results, and you will lose less data. If your tracks remain noisy at low speeds, try non-zero values here. Please note, some measurements may not report any speed value at all (some network-based methods), in which case you would not record anything.</string>
```

Comment:
```suggestion
    <string name="monitoring_min_speed_descr_recommendation">Try increasing the value required to detect motion in the \"Minimum logging displacement\" filter (B) first. This could end up recording nothing with some network-based methods, because there is no reported speed value.</string>
```